adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -159,8 +165,41 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
         sc.loadFastq(args.inputPath, Option(args.pairedFastqFile), Option(args.fastqRecordGroup), stringency)
       } else if (args.forceLoadIFastq) {
         sc.loadInterleavedFastq(args.inputPath)
-      } else if (args.forceLoadParquet) {
-        sc.loadParquetAlignments(args.inputPath)
+      } else if (args.forceLoadParquet ||
+        args.limitProjection ||
+        args.useAlignedReadPredicate) {
```

Comment:
Agreed, not a bad idea. I've added logging for this at https://github.com/bigdatagenomics/adam/pull/844/files#diff-1ee985bcfcfad885cfc143944371af07R161.